pkg/kv/kvserver/metrics.go

```diff
@@ -2607,9 +2607,9 @@ func newStoreMetrics(histogramWindow time.Duration) *StoreMetrics {
 		// L0SublevelsMax. this is not exported to as metric.
 		sm.l0SublevelsTracker.swag = slidingwindow.NewMaxSwag(
```

Comment:
Should we get rid of this? It's only used for metrics, AFAICT, and is surfacing a value we're no longer using in production code.

Reply:
The gossiped value is taken from this. It isn't exposed to metrics. We can remove it in 23.2.